app/src/main/java/fr/gaulupeau/apps/Poche/ui/ArticleListsFragment.java

```diff
@@ -216,4 +217,33 @@ private void updateList(int position, boolean forceContentUpdate) {
         }
     }
 
+    public void scroll(boolean up) {
+
+        LinearLayoutManager listLayout = getCurrentFragment().recyclerViewLayoutManager;
```

Comment:
A null-check for both `getCurrentFragment()` and `recyclerViewLayoutManager` wouldn't hurt here.

Reply:
Done in the last commit.